js/src/shell/OSObject.cpp

```diff
@@ -612,6 +666,11 @@ static const JSFunctionSpecWithHelp osfile_functions[] = {
 "  as the second argument, in which case it returns a Uint8Array. Filename is\n"
 "  relative to the current working directory."),
 
+    JS_FN_HELP("listDir", osfile_listDir, 1, 0,
+"listDir(filename)",
+"  Read entire contents of a directory. Returns a list of filenames within the given\n"
+"  directory. Filename is relative to the current working directory."),
```

Comment:
This sounds like the "filename" in the returned list is relative to the CWD.
How about "Filename parameter is relative to ..." ?

Also, this should mention that the list contains directories and "." and ".."